### PR TITLE
Add support for extra sources

### PR DESCRIPTION
**Pull request by justin-stephenson**

### CODE REVIEW

**Comment by justin-stephenson** [comment]
`Test / action (pull_request)` failing due to `next-actions/build-srpm@master` in workflow file

**Comment by ikerexxe** [comment]
Is a tarball the easiest way to add a bunch of files in rpm generation?

**Comment by pbrezina** [comment]
I agree, it would be nice to be able to just specify the file. If we can't pass list of files (I hope we can) we can use some separator.

**Comment by pbrezina** [comment]
> `Test / action (pull_request)` failing due to `next-actions/build-srpm@master` in workflow file

I think you can modify it to call the action from checkouted repository instead from tag. E.g. https://github.com/next-actions/copr/blob/master/.github/workflows/test.yml#L83

**Comment by justin-stephenson** [comment]
> I agree, it would be nice to be able to just specify the file. If we can't pass list of files (I hope we can) we can use some separator.

I couldn't see how to pass a list of files, so I went with the tarball approach. Can you expand on `we can use some separator` ?

**Comment by pbrezina** [comment]
See `files` in https://github.com/next-actions/print-logs#example-usage for example.

Code: https://github.com/next-actions/print-logs/blob/master/action.yml#L21

**Comment by justin-stephenson** [comment]
> > `Test / action (pull_request)` failing due to `next-actions/build-srpm@master` in workflow file
> 
> I think you can modify it to call the action from checkouted repository instead from tag. E.g. https://github.com/next-actions/copr/blob/master/.github/workflows/test.yml#L83

Thanks, I added a separate commit to make this change.

**Comment by justin-stephenson** [comment]
> See `files` in https://github.com/next-actions/print-logs#example-usage for example.
> 
> Code: https://github.com/next-actions/print-logs/blob/master/action.yml#L21

Done, please check.

**Review by ikerexxe — APPROVED** [review]
The new version certainly looks better. Thanks for taking care of it.

**Comment by pbrezina** [comment]
LGTM, can you also extend the PR CI check to list files inside src.rpm and check that all source are there?

**Comment by justin-stephenson** [comment]
> LGTM, can you also extend the PR CI check to list files inside src.rpm and check that all source are there?

PR updated.

File: .github/workflows/test.yml
Comment (pbrezina) [inline .github/workflows/test.yml:57]: Can you put this to the example in the readme file? After that, feel free to merge it.

**Review by pbrezina — CHANGES_REQUESTED** [review]
Thank you. Last nitpick and let's merge it.

File: .github/workflows/test.yml
Comment (justin-stephenson) [reply .github/workflows/test.yml:57]: Done. I don't have merge access

**Pull request merged**
